README.md

```diff
@@ -216,6 +216,7 @@ Reporters behave as described at: https://github.com/shyiko/ktlint
 The `experimentalRules` property enables rules which are part of ktlint's experimental rule set.
 
 The `fileBatchSize` property configures the number of files that are processed in one Gradle Worker API call.
+Default value is 30 for linting and 20 for formatting.
```

Comment:
It is a bit strange to have one config property that changes two different defaults. Do you think the needs are different enough they need two separate values? I'd be in favor or one param and one default if there's a reasonable shared value that's close enough to optimal for both linting and formatting -- mostly for the sake of simplicity since I doubt most people will need to tune it to that level of detail.

Reply:
Ok, i'll change it back to one default. And if needed, people can still set the value explicitely.